### PR TITLE
Update sv.json & added translation in ContactList.tsx

### PR DESCRIPTION
**Pull request by MagnusLundqvist**
**Pull Request Description**
Updated sv.json with missing translations.
Updated sv.json with some minor changes in translation.
Replaced Twitter -> Life Invader, aswell as all "Tweet" & "Retweet" translations in sv.json.
Renamed "APPS_MARKETPLACE" from Blocket -> Marknadsplatsen. (Just to not use real life company names.)
Added "My Number" translation in ContactList.tsx
Added "Nearby Share" translation in ContactList.tsx
Added  "MY_NUMBER" aswell as "NEARBY_SHARE" in en.json and sv.json, for easy translations or renaming.

**Pull Request Checklist**:
* [x] Have you followed the guidelines in our Contributing document and Code of Conduct?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/project-error/npwd/pulls) for the same update/change?
* [x] Have you built and tested NPWD in-game after the relevant change?

### CODE REVIEW

File: apps/phone/src/locale/sv.json [hunk 1]
```diff
@@ -298,14 +312,17 @@
...
       "TITLE": "Titel",
-      "SEND": "Skicka",
       "CONTENT": "Innehåll",
       "SHARE": "Dela",
+      "ADD": "Lägg till",
+      "SEND": "Skicka",
       "WRITE_TO_CLIPBOARD_TOOLTIP": "Kopiera {{ content }}",
-      "WRITE_TO_CLIPBOARD_MESSAGE": "Kopierade {{content}} till urklipp!"
+      "WRITE_TO_CLIPBOARD_MESSAGE": "Kopierade {{content}} till urklipp!",
+      ""
```
Comment (derletzteritter) [inline apps/phone/src/locale/sv.json:321]: Empty string?

**Review by derletzteritter — CHANGES_REQUESTED** [review]
Looks like there are some syntax errors in the json files you've edited :)

File: apps/phone/src/locale/sv.json (hunk 1 above)
Comment (MagnusLundqvist) [reply apps/phone/src/locale/sv.json]: Sorry fixed it now :)

**Comment by derletzteritter** [comment]
LGTM! Thank you for your contribution!

**Pull request merged**